Google.Api.Gax.Grpc/ApiCall.cs

```diff
@@ -151,12 +152,19 @@ public sealed class ApiCall<TRequest, TResponse>
         internal ApiCall<TRequest, TResponse> WithMergedBaseCallSettings(CallSettings callSettings) =>
             new ApiCall<TRequest, TResponse>(_asyncCall, _syncCall, callSettings.MergedWith(BaseCallSettings));
 
-        internal ApiCall<TRequest, TResponse> WithRetry(IClock clock, IScheduler scheduler) =>
+        // Note: it seems unfortunate that we can't reuse whatever logger is already configured here, but we don't
```

Comment:
But when we configure  them we are effectively using the same one, right?

Reply:
Yes, that's right.